cli/foundation-models/system/inference/utils/base64_to_jpeg.py

```diff
@@ -0,0 +1,45 @@
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Read base64 encoded image from txt file and convert it to image file
+
+import argparse
+import re
+import json
+import io
+import base64
+from PIL import Image
+
+
+INPUT_PROMPT_COLUMN = "prompt"
+OUTPUT_IMAGE_COLUMN = "generated_image"
+
+def base64_str_to_image(response_file: str):
+    """
+    Read file that contains response from online endpoint. Extarct base64 encoded images and save them as image files.
+
+    :param response_file: Path to json file, which has response received from endpoint.
+    :type response_file: str
+    :return: None
+    """
+    with open(response_file) as f:
+        serialized_image_json = f.read().strip()
+
+    serialized_image_json=serialized_image_json.replace("\\\"","\"")[1:-1] 
```

Comment:
Better is to use  "-o json> response_file.json" in invoke endpoint command to store output in json and then read json as it using json.load

Reply:
I tried it gets saved to file extra quotes around the json string and within the json text, it has single quotes, this causes json.loads to fail. Hence the longer approach.